### ISSUE TITLE
Resizing tables at runtime

### ISSUE THREAD

**Issue by stdword** [sethyuan/logseq-plugin-another-embed#17]
I've made some changes for tables resizing:
1. Changing certain column's width affects all others columns (and all `col-w-X` props should be saved)
2. Resizing at runtime (to see results immediately)

## Demo
|Before|After|
|:---:|:---:|
|<img src="https://github.com/sethyuan/logseq-plugin-another-embed/assets/1984175/0be0d7c8-af2d-4a54-abfd-9b7cddc2a20f"/>|<img src="https://github.com/sethyuan/logseq-plugin-another-embed/assets/1984175/7fd35b36-5a1c-4611-8a1f-b9f3597650d6"/>|

This solution based on [logseq-custom-files](https://github.com/cannibalox/logseq-custom-files#query-table-view-resizer-). But I can create a PR with its integration to this plugin. What do you think? Should I do that?

**Comment by sethyuan** [comment]
Thanks for the effort, I think 2 is great to have, but I have some concerns about 1, if all columns width are also persisted, the table will lose its own auto column width algorithm when further adjusting a column.

**Comment by stdword** [comment]
Yes, and this is an idea: if user starts to adjust the columns — the auto adjusting doesn't required any more for particular table.

These points only make sense (to me) together.

**Comment by sethyuan** [comment]
I don't think so, say your viewport is 800px now, after fixing the width of one column, you enlarged the viewport width to 1200px (via `t w` perhaps), with the auto adjusting behavior of the table itself, it will display OK, but if all columns are assigned a width then it will not display OK.

**Comment by stdword** [comment]
This is interesting: when width of ALL columns set to fixed pixels — its sizes changes relatively to whole table width. Just like none of the columns are fixed-width:

<img width="800px" src="https://github.com/sethyuan/logseq-plugin-another-embed/assets/1984175/1f02cae1-f2a4-4b40-8140-5fb8f1097301"/>

**Comment by sethyuan** [comment]
Oh my! But this defeats the purpose of setting a fixed width to a column, isn't it?

**Comment by stdword** [comment]
No: we are adjusting column width with respect to each other. A proportion, not a pixels. It is better to use % instead of px.

**Comment by sethyuan** [comment]
> No: we are adjusting column width with respect to each other. A proportion, not a pixels. It is better to use % instead of px.
> 
> Well, I realize, it is a bit another feature.

Agreed!
